AvatarGenerator/po/it-local.po

```diff
@@ -0,0 +1,161 @@
+# Addon AvatarGenerator for Gramps.
+# Copyright (C) 2025 Gramps
+# This file is distributed under the same license as the gramps package.
+#
+#
+# Sebastiano Pistore <SebastianoPistore.info@protonmail.ch>, 2025.
+msgid ""
+msgstr ""
+"Project-Id-Version: it\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-06-14 11:49+1000\n"
+"PO-Revision-Date: 2025-01-04 17:35+0100\n"
+"Last-Translator: Sebastiano Pistore <SebastianoPistore.info@protonmail.ch>\n"
+"Language-Team: \n"
+"Language: it_IT\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Poedit-SourceCharset: utf-8\n"
+"X-Generator: Poedit 3.5\n"
+
+#: AvatarGenerator/AvatarGenerator.gpr.py:23
+msgid "Avatar Generator Tool"
+msgstr "Avatar Generator Tool"
+
+#: AvatarGenerator/AvatarGenerator.gpr.py:24
+msgid "A tool to add avatar pictures to your family tree"
+msgstr "Strumento per aggiungere avatar alle persone senza alcuna immagine che le rappresenti"
+
+#: AvatarGenerator/AvatarGenerator.py:58
+msgid "male"
+msgstr "maschio"
+
+#: AvatarGenerator/AvatarGenerator.py:58
+msgid "female"
+msgstr "femmina"
+
+#: AvatarGenerator/AvatarGenerator.py:58
+msgid "unknown"
+msgstr "sconosciuto"
+
+#: AvatarGenerator/AvatarGenerator.py:59
+msgid "Option"
+msgstr "Opzioni"
+
+#: AvatarGenerator/AvatarGenerator.py:59
+msgid "single image mode"
+msgstr "immagine singola"
+
+#: AvatarGenerator/AvatarGenerator.py:65
+msgid "Category"
+msgstr "Categoria"
+
+#: AvatarGenerator/AvatarGenerator.py:66
+msgid "Choose how many images you'd like to use."
+msgstr "Scegliete quante immagini utilizzare."
```

Comment:
The standard is used in the previous gramps translations (not chosen by me) for descriptive text like this (outside the menu items, where a more direct form is used) is the impersonal one with the verb in the infinite form , so please change this to "Scegliere"

Reply:
You're right.